### ISSUE TITLE
Add webpack, babel, and eslint

### ISSUE THREAD

**Issue by deadbirdi** [anshumanv/evolutions-webapp#11]
I'm adding webpack, babel, and eslint for compilation 
---
##### This issue is about {Add webpack, babel, and eslint}

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by Nitiquita** [comment]
@sneakycr0w is this up for grabs?

**Comment by anshumanv** [comment]
@Nitiquita thanks for showing interest here, PR has already been made for this, meanwhile you can look at the existing issues. Thanks 👍
